src/Microsoft.AspNet.Authentication/DataHandler/TicketDataFormat.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.AspNet.Authentication
 {
     public class TicketDataFormat : SecureDataFormat<AuthenticationTicket>
     {
-        public TicketDataFormat(IDataProtector protector) : base(new TicketSerializer(), protector)
+        public TicketDataFormat(IDataProtector protector)
+            : base(TicketSerializer.Default, protector)
```

Comment:
I think we addressed that by just newing up a new instance instead of using the static when we removed them in the current bits.